benchmark/ts/strip-types.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const path = require('path');
+
+const js = path.resolve(__dirname, '../fixtures/strip-types-benchmark.js');
+const ts = path.resolve(__dirname, '../fixtures/strip-types-benchmark.ts');
+
+const bench = common.createBenchmark(main, {
+  type: ['ts', 'js'],
+  n: [1e4],
+}, {
+  flags: ['--experimental-strip-types', '--no-warnings'],
```

Comment:
Since --transform-types is an implicit enable of --strip-types, for comparison reasons would be better if both exist in the same benchmark file.

Reply:
I'm not sure what you mean, you mean create one benchmark file?